src/shared/components/header/index.js

```diff
@@ -7,17 +7,26 @@ import DesktopNavbar from 'shared/components/navbar/desktop'
 import MobileNavbar from 'shared/components/navbar/mobile'
 import styles from './index.module.css'
 
+const defaultScrollOptions = { offset: -66, align: 'top', duration: 800 }
+
 class Header extends Component {
   constructor () {
     super()
 
     this.state = {
-      isSticky: false
+      isSticky: false,
+      gettingStartedElement: undefined
     }
   }
 
+  componentDidMount () {
+    this.scrollToComponent = require('react-scroll-to-component')
+    this.setState({
+      gettingStartedElement: document.getElementById('gsContainer')
```

Comment:
Since `gettingStartedElement` isn't used in the render flow, it should be set into a property of the class instead of the state.

Reply:
Another thing: accessing elements by id is not react idiomatic. We should use refs & callbacks to pass the node around. We can keep it like this, but please create an issue to improve that later on.

Reply:
I needed the "getting started" div ref on `header` component. So, I was trying to do that using refs & callbacks in my `layout.js` (parent component). However, when I try to pass a simple prop to `children()` function, it breaks down. That happens because the function is not doing the spread of `layout props` (that were the props that I was trying to pass) and `route props`. They have something like: 
```
layoutProps ? layoutProps : routeProps
```
So, since it needs `routeProps`, when I pass layout props, they're replaced and it breaks.
For that reason, I was not able to do this that way. I don't know if there's another workaround, but we can check it later.